fearless/Common/Network/WalletNetworkFacade+Protocol.swift

```diff
@@ -4,7 +4,7 @@ import RobinHood
 import IrohaCrypto
 import BigInt
 
-extension WalletNetworkFacade: WalletNetworkOperationFactoryProtocol {
+extension WalletNetworkFacade: WalletNetworkOperationFactoryProtocol, RuntimeConstantFetching {
```

Comment:
I suggest to move conformance of RuntimeConstantFetching to `WalletNetworkFacade+MinimalBalance`